creativecloud/blocks/cc-mobile-app-banner/cc-mobile-app-banner.js

```diff
@@ -0,0 +1,69 @@
+import { getConfig } from '../../scripts/utils.js';
+
+async function getKey(product) {
+  const config = getConfig();
+  let keyMatch = [];
+  const resp = await fetch(`${config.contentRoot ?? ''}/branch-io-key.json`);
+  if (resp.ok) {
```

Comment:
if response return is not ok we are going to get an erro rin the later return `return keyMatch[0]?.key;`
Can we do an early return here? 
if (!resp.ok) return null;

Reply:
I think it'll return undefined if the resp is not ok and init method we handle. So I believe that it will get handled